README.md

```diff
@@ -128,37 +128,40 @@ app to Slack hosting using `slack deploy`:
 $ slack deploy
 ```
 
-After deploying, [create a new link trigger](#create-a-link-trigger) for the
-production version of your app (not appended with `(dev)`). Once the trigger is
-invoked, the workflow should run just as it did in when developing locally.
+After deploying, you'll be prompted to
+[create a new link trigger](#creating-triggers) for the production version of
+your app. When that trigger is invoked, the workflow should run just as it did
+in when developing locally (but without requiring your server to be running).
```

Comment:
Just me or is the wording in this sentence a bit weird? "Just as it did in when developing locally..."

Reply:
The 'in' may be superfluous; `Just as it did when developing locally...`

Reply:
Totally right, great catch, will edit!